packages/wdio-browserstack-service/src/service.js

```diff
@@ -36,16 +44,18 @@ export default class BrowserstackService {
             this.sessionBaseUrl = 'https://api-cloud.browserstack.com/app-automate/sessions'
         }
 
+        this.scenariosThatRan = []
+
         return this._printSessionURL()
     }
 
-    afterSuite(suite) {
-        if (Object.prototype.hasOwnProperty.call(suite, 'error')) {
-            this.failures++
-        }
+    beforeFeature(uri, feature) {
+        const oldTitle = this.fullTitle
+        this.fullTitle = [this.fullTitle, feature.document.feature.name].filter(Boolean).join(', ')
```

Comment:
using shadow of  `this.fullTitle` make it hard to read, replace with oldTitle where possible